server/internal/mcpmetadata/impl.go

```diff
@@ -284,10 +284,9 @@ func (s *Service) ServeInstallPage(w http.ResponseWriter, r *http.Request) error
 		return oops.E(oops.CodeUnexpected, err, "failed to describe toolset").Log(ctx, s.logger)
 	}
 
-	var (
-		logoAssetURL string
-		docsURL      string
-	)
+	logoAssetURL := os.Getenv("GRAM_SITE_URL") + "/external/sticker-logo.png"
```

Comment:
Do not access environment variables outside of `cmd/`. Instead pass in the data your service needs when constructing it. This limits implicit behavior in services and makes them easier to test. A service that looks up env vars contains implicit behavior.

Reply:
Sorry. this was absolute amateur hour.